src/concepts/choosing_a_dataset_mobile.md

```diff
@@ -24,15 +28,15 @@ Klar is the known release name for Focus in Germany.
 
 For more information on how telemetry is sent for iOS apps, see the [telemetry documentation][ios].
 
-Some telemetry is also sent by FirefoxReality and some non-Mozilla forks of our
-browsers. It is recommended that you filter on App Name to make sure that you are looking at only the app for which you want to analyze data.
+Some telemetry is also sent by non-Mozilla forks of our browsers.
+When consulting legacy telemetry, filter on app name to make sure that you are looking at only the app for which you want to analyze data.
 
 [fxlite]: https://github.com/mozilla-tw/FirefoxLite/blob/master/docs/telemetry.md
 [ios]: https://github.com/mozilla-mobile/telemetry-ios
 
-## Raw Pings
+## Legacy ping tables
 
-Mobile data is structured differently when compared to desktop data. Instead of sending a `main` ping, mobile has provides the following key types of pings:
+Legacy (pre-Glean) mobile data is structured differently than desktop data. Instead of sending a `main` ping, mobile has provides the following key types of pings:
```

Comment:
I think the info here on legacy `event` pings is incorrect. It's my understanding that events for mobile apps are sent in `mobile-event` and `focus-event` pings. The `event` ping is Desktop-specific.